gcloud-java-logging/src/main/java/com/google/gcloud/logging/spi/v2/ConfigServiceV2Api.java

```diff
@@ -0,0 +1,533 @@
+/*
+ * Copyright 2015 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+/*
+ * EDITING INSTRUCTIONS
+ * This file was generated from the file
+ * https://github.com/google/googleapis/blob/master/google/logging/v2/logging_config.proto
+ * and updates to that file get reflected here through a refresh process.
+ * For the short term, the refresh process will only be runnable by Google engineers.
+ * Manual additions are allowed because the refresh process performs
+ * a 3-way merge in order to preserve those manual additions. In order to not
+ * break the refresh process, only certain types of modifications are
+ * allowed.
+ *
+ * Allowed modifications - currently these are the only types allowed:
+ * 1. New methods (these should be added to the end of the class)
+ * 2. New imports
+ * 3. Additional documentation between "manual edit" demarcations
+ *
+ * Happy editing!
+ */
+package com.google.gcloud.logging.spi.v2;
+
+import com.google.logging.v2.ConfigServiceV2Grpc;
+import com.google.logging.v2.CreateSinkRequest;
+import com.google.logging.v2.DeleteSinkRequest;
+import com.google.logging.v2.GetSinkRequest;
+import com.google.logging.v2.ListSinksRequest;
+import com.google.logging.v2.ListSinksResponse;
+import com.google.logging.v2.LogSink;
+import com.google.logging.v2.UpdateSinkRequest;
+import com.google.protobuf.Empty;
+import io.gapi.gax.grpc.ApiCallable;
+import io.gapi.gax.grpc.PageDescriptor;
+import io.gapi.gax.grpc.ServiceApiSettings;
+import io.gapi.gax.internal.ApiUtils;
+import io.gapi.gax.protobuf.PathTemplate;
+import io.grpc.ManagedChannel;
+import java.io.IOException;
+
+// Manually-added imports: add custom (non-generated) imports after this point.
+
+
+
+// AUTO-GENERATED DOCUMENTATION AND SERVICE - see instructions at the top of the file for editing.
+/**
+ * See //google/logging/v2/logging.proto for documentation
+ *
+ * <!-- manual edit -->
+ * <!-- end manual edit -->
+ */
+@javax.annotation.Generated("by the veneer generator")
+public class ConfigServiceV2Api implements AutoCloseable {
```

Comment:
I think the scope of #472 is too wide. I assume that it is for using a cloud emulator (once provided) instead of the local fake implementation. If you plan to defer writing tests for the spi layer until then create a separate issue for it and make it depends on #472.